### DISCUSSION TITLE
How do I show a complete album, given a title in the playlist?

### DISCUSSION THREAD

**Discussion by smurfix** [jcorporation/myMPD#1020] [Q&A] [answered]
Hello,

when the playlist contains a title on an album that has multiple artists … how do I get to the (complete) album?

Title > menu > Album > Details shows the album but filters it by the artist. The other titles aren't shown.

**Comment by jcorporation** [comment, marked as answer]
You must enable the AlbumArtist tag.

**Comment by smurfix** [comment]
Where do I find that? I only see separate "Album" and "Artist" tags on the relevant pages, and they're all on.

**Comment by jcorporation** [comment]
You must enable it in MPD first.
